e2e-tests/cypress/support/insurance/check-your-business-summary-list.js

```diff
@@ -143,7 +143,7 @@ const checkYourBusinessSummaryList = ({
     const fieldId = COMPANY_SIC;
 
     const { expectedKey } = getSummaryListField(fieldId, FIELDS);
-    const expectedValue = application.EXPORTER_COMPANY[fieldId][0];
+    const expectedValue = `${application.EXPORTER_COMPANY[fieldId][0]} - Financial intermediation not elsewhere classified`;
```

Comment:
Constant?

Reply:
we haven't stored anything for this company in constants in this test.  Should I leave it as is?